test/test_fx.py

```diff
@@ -2260,5 +2262,342 @@ def test_get_torch_func_signature_exhaustive(self, device, dtype, op):
+    return run_test
+
+
+def generate_tests(test_cls, constructor_arg_db):
+    # test all modules underneath these namespaces...
+    NAMESPACES = [
+        torch.nn,
+        torch.nn.qat,
+        torch.nn.quantizable,
+        torch.nn.quantized,
+    ]
+    # ...except these
+    MODULES_TO_SKIP = {
+        torch.nn.Module,  # no forward() implemented
+        torch.nn.Container,  # deprecated
+        torch.nn.NLLLoss2d,  # deprecated
+        torch.nn.quantized._ConvNd,  # base class in __all__ for some reason
+        torch.nn.quantized.QFunctional,  # not intended to use forward()
+        torch.nn.quantized.FloatFunctional,  # not intended to use forward()
+        torch.nn.quantized.FXFloatFunctional,  # not intended to use forward()
+        torch.nn.RNNBase,  # abstract base class
+        torch.nn.RNNCellBase,  # abstract base class
+        torch.nn.ParameterList,  # no forward() implemented
+        torch.nn.ParameterDict,  # no forward() implemented
+        torch.nn.ModuleList,  # no forward() implemented
+        torch.nn.ModuleDict,  # no forward() implemented
+    }
+    # these modules requires FBGEMM backend to instantiate
+    MODULES_THAT_REQUIRE_FBGEMM = {
+        torch.nn.quantized.Conv1d,
+        torch.nn.quantized.Conv2d,
+        torch.nn.quantized.Conv3d,
+        torch.nn.quantized.ConvTranspose1d,
+        torch.nn.quantized.ConvTranspose2d,
+        torch.nn.quantized.ConvTranspose3d,
+        torch.nn.quantized.Linear,
+    }
+    # these modules currently cannot be symbolically traced (yet)
+    UNTRACEABLE_MODULES = {
+        torch.nn.AdaptiveLogSoftmaxWithLoss,
+        torch.nn.BatchNorm1d,
+        torch.nn.BatchNorm2d,
+        torch.nn.BatchNorm3d,
+        torch.nn.ConvTranspose1d,
+        torch.nn.ConvTranspose2d,
+        torch.nn.ConvTranspose3d,
+        torch.nn.CrossMapLRN2d,
+        torch.nn.GRU,
+        torch.nn.InstanceNorm1d,
+        torch.nn.InstanceNorm2d,
+        torch.nn.InstanceNorm3d,
+        torch.nn.LSTM,
+        torch.nn.LSTMCell,
+        torch.nn.LazyBatchNorm1d,
+        torch.nn.LazyBatchNorm2d,
+        torch.nn.LazyBatchNorm3d,
+        torch.nn.LazyConvTranspose1d,
+        torch.nn.LazyConvTranspose2d,
+        torch.nn.LazyConvTranspose3d,
+        torch.nn.RNN,
+        torch.nn.Softmax2d,
+        torch.nn.SyncBatchNorm,
+        torch.nn.Transformer,
+        torch.nn.TripletMarginWithDistanceLoss,
+        torch.nn.qat.Conv2d,
+        torch.nn.qat.Conv3d,
+        torch.nn.qat.Linear,
+        torch.nn.quantizable.LSTM,
+        torch.nn.quantizable.LSTMCell,
+        torch.nn.quantizable.MultiheadAttention,
+        torch.nn.quantized.Conv1d,
+        torch.nn.quantized.Conv2d,
+        torch.nn.quantized.Conv3d,
+        torch.nn.quantized.ConvTranspose1d,
+        torch.nn.quantized.ConvTranspose2d,
+        torch.nn.quantized.ConvTranspose3d,
+        torch.nn.quantized.ELU,
+        torch.nn.quantized.Hardswish,
+    }
```

Comment:
It might also be worth adding comments after each of these showing why they don't work. I dumped this out programmatically from the test: https://gist.github.com/jamesr66a/63c01b176a88a04916bdb21862947247

Reply:
Added!